### PR TITLE
Update README

### LINKED ISSUE #130: Review and update README
Review and update the dojo/meta README. For example, we've moved on from RxJS Observables to just shim the ES8 spec, etc. So let's review the content and make sure it reflects our current thinking.

### PR DESCRIPTION
**Pull request by agubler**
Update the README with the relevant package statuses, remove maintainers and update the npm badges since moving to scoped package `@dojo`

Resolves #130

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,11 +41,14 @@ There are several packages that makeup the Dojo 2 platform:
 
 We also have packages that are still in their planning stages and as such might not be part of the initial Dojo 2 release.
 
+* [`dojo/actions`](https://github.com/dojo/actions) - A command like library for Dojo 2 applicationsz
```
Comment (dylans) [inline README.md]: extra z at the end here.

File: README.md [hunk 2]
```diff
@@ -47,11 +41,14 @@ There are several packages that makeup the Dojo 2 platform:
...
+* [`dojo/app`](https://github.com/dojo/app) - An application framework for Dojo 2
 * [`dojo/crypto`](https://github.com/dojo/crypto) - A set of libraries to deal with cryptography
+* [`dojo/dataviz`](https://github.com/dojo/dataviz) - A data vizualization library for Dojo 2
 
+We have some deprecated packages:
 
-We have deprecated a package:
-
+* [`dojo/dom`](https://github.com/dojo/dom) - A set of APIs for manipulating the DOM
```
Comment (dylans) [inline README.md]: is this deprecated... I was thinking it might still be useful for someone migrating from 1.x to 2? Maybe not.

File: README.md [hunk 3]
```diff
@@ -72,54 +69,59 @@ These packages are aimed at Dojo 2 contributors rather than Dojo 2 end users.
...
+#### Proposed packages
 
-#### CLI Packages
+|Package|CI Status|Code Coverage|
+|-------|---------|-------------|
+|[dojo/actions](https://github.com/dojo/actions)|[![Build Status](https://travis-ci.org/dojo/actions.svg?branch=master)](https://travis-ci.org/dojo/actions)|[![codecov.io](https://codecov.io/gh/dojo/actions/branch/master/graph/badge.svg)](https://codecov.io/gh/dojo/actions/branch/master)|
+|[dojo/app](https://github.com/dojo/app)|[![Build Status](https://travis-ci.org/dojo/app.svg?branch=master)](https://travis-ci.org/dojo/app)|[![codecov.io](https://codecov.io/gh/dojo/app/branch/master/graph/badge.svg)](https://codecov.io/gh/dojo/app/branch/master)|
+|[dojo/crypto](https://github.com/dojo/crypto)| | | |
+|[dojo/dataviz](https://github.com/dojo/dataviz)|[![Build Status](https://travis-ci.org/dojo/dataviz.svg?branch=master)](https://travis-ci.org/dojo/dataviz)| | |
+|[dojo/dom](https://github.com/dojo/dom)|[![Build Status](https://travis-ci.org/dojo/dom.svg?branch=master)](https://travis-ci.org/dojo/dom)|[![codecov.io](https://codecov.io/gh/dojo/dom/branch/master/graph/badge.svg)](https://codecov.io/gh/dojo/dom/branch/master)|
```
Comment (dylans) [inline README.md]: Proposed or deprecated?

Comment (agubler) [inline README.md]: It's a fair point, @rishson said to put it here but i realise that is inconsistent with the statement above.

File: README.md (hunk 2 above)
Comment (agubler) [inline README.md]: I think for now it is deprecated in it current form. It was written pre typescript 1.8 (even though it has been updated through to 2) and is not used anywhere within the current dojo 2 packages. Would be happy to revisit once we have some use cases?

File: README.md (hunk 1 above)
Comment (agubler) [inline README.md]: we like the zzzzzzzzzz's (will fix up)

File: README.md (hunk 3 above)
Comment (rishson) [inline README.md]: Lets move to 'deprecated' then - to be consistent.

**Comment by agubler** [comment]
@dylans have moved `dom` to proposed/planned packages.

**Pull request merged**